website/docs/getting-started/setup/installation-guides/kubernetes/README.mdx

```diff
@@ -310,59 +310,89 @@ Follow the steps below to install Appsmith on your Kubernetes cluster:
+    ```
+4. Edit the `values.yaml` file and set the following parameters
+
+   ```yaml
+    applicationConfig:
+      APPSMITH_ENCRYPTION_PASSWORD: "<PASSWORD>"
+      APPSMITH_ENCRYPTION_SALT: "<SALT_PASSWORD>"
+      APPSMITH_LICENSE_KEY: "<LICENSE_KEY>"
+   ```
 
 <details id="highAvailability" open>
   <summary> <b>To configure High Availability (HA), be sure to configure the following</b> </summary>
 
+:::note Important
+
+  Both PostgreSQL and MongoDB must be enabled or configured to point to an external instance when `autoscaling` is enabled
+:::
+
   1. Install `metrics-server`, which provides vital metrics to the Horizontal Pod Autoscaler (HPA) to scale the pods.
 
     ```bash
     kubectl apply -f https://github.com/kubernetes-sigs/metrics-server/releases/latest/download/components.yaml
     ```
-  2. Get the initial `values-ee.yaml`.
-
-    ```bash
-    helm show values appsmith-ee/appsmith --version 2.0.0 > values-ee.yaml
-    ```
-
-  3. PostgreSQL has to be enabled or an external PostgreSQL URL has to be provided.
-
-   ```yaml
-         postgresql:
-         # highlight-next-line
-            enabled: true
-            auth:
-              username: root
-  ```
 
-  4.  Enable `autoscaling` in the `values-ee.yaml`:
+  2. Enable `autoscaling` in the `values.yaml`. The below shows the `autoscaling` configuration for the Appsmith Deployment:
 
    ```yaml
         autoscaling:
           # highlight-next-line
           enabled: true
           minReplicas: 2
           maxReplicas: 2
-  ```
+   ```
 
-  5.  Enable `mongodb` in the `values-ee.yaml`, or point it to an [external MongoDB](/getting-started/setup/instance-configuration/custom-mongodb-redis#custom-mongodb).
+  3. PostgreSQL has to be enabled Or configure an external PostgreSQL in the `applicationConfig` section in `values.yaml`.
 
-  ```yaml
-    mongodb:
-      # highlight-next-line
-      enabled: true
-      service:
-        nameOverride: appsmith-mongodb
-  ```
+      i. Configure the parameters to enable PostgreSQL as shown below:
 
-6. Create a shared file system. This is required for some Appsmith features that make use of the filesystem, like `git`-connected applications. Read more about this in the [Create a shared file system](#create-a-shared-file-system) section.
+      ```yaml
+            # enable PostgreSQL
+            postgresql:
+            # highlight-next-line
+                enabled: true
+                auth:
+                  username: root
+      ```
 
-7. Set a secret value to password, and to salt.
+      ii. Configure the parameters to point to an external PostgreSQL as shown below:
+    
+      ```yaml
+        # configure an external PostgreSQL
+        applicationConfig::
+          APPSMITH_KEYCLOAK_DB_DRIVER: "postgresql"
+          APPSMITH_KEYCLOAK_DB_USERNAME: "<POSTGRES_USER>"
+          APPSMITH_KEYCLOAK_DB_PASSWORD: "<POSTGRES_PASSWORD>"
+          APPSMITH_KEYCLOAK_DB_URL: "<POSTGRES_DB_URL>"
+      ```
```

Comment:
Same as above, change `ii.` to `2.`, and the alignment of the code block is also off.